tox.ini

```diff
@@ -1,32 +1,29 @@
 [tox]
 envlist=
-    coverage-erase,
+    flake8,
     py35,
     py36,
     py37,
     py38,
     pypy3,
-    coverage-report
+    coverage,
 
 [testenv]
 deps = pytest
-       pytest-flake8
-       coverage
 commands =
-    !azure: coverage run --append {envbindir}/pytest --junitxml=junit-{envname}.xml --flake8 {posargs:test_plugin.py} []
-    azure: {envbindir}/pytest --junitxml=junit-{envname}.xml --flake8 {posargs:test_plugin.py} []
+    {envbindir}/pytest {posargs:test_plugin.py} []
 
-[testenv:coverage-erase]
-deps = coverage
-skip_install = true
+[testenv:coverage]
+basepython = python3
+deps =
+    {[testenv]deps}
+    pytest-cov
+    coverage-python-version
 commands =
-    coverage erase
-
+    {envbindir}/py.test {posargs:test_plugin.py} --cov-report=html []
```

Comment:
This line seems to miss a `--cov=` option so maybe coverage is not executed at all.
As there is no `src` directory maybe it helps to switch back to the `coverage run` style used before in `testenv`.